aiopolly/__init__.py

```diff
@@ -12,5 +12,5 @@
     'utils'
 ]
 
-__version__ = '0.1.1'
+__version__ = '0.2.1'
 __api_date__ = datetime.date(2018, 8, 2)  # August 2, 2018
```

Comment:
Thanks for your contribution, I guess we should also make just this tiny change: https://docs.aws.amazon.com/polly/latest/dg/doc-history.html

```suggestion
__api_date__ = datetime.date(2019, 7, 30)  # July 30, 2019
```